app/services/webhooks/add_on_service.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Webhooks
+  class AddOnService < Webhooks::BaseService
+    private
+
+    def current_organization
+      @current_organization ||= object.organization
+    end
+
+    def object_serializer
+      ::V1::InvoiceSerializer.new(
+        object,
+        root_name: 'invoice',
+        includes: %i[customer subscription],
+      )
+    end
+
+    def webhook_type
+      'add_on.created'
```

Comment:
Since the the webhook is sending an invoice object, I suggest to set `webhook_type` to `invoice.add_on_added`

Reply:
changed